.circleci/config.yml

```diff
@@ -138,19 +138,19 @@ jobs:
 
   helicsInstall1:
     docker:
-      - image: helics/buildenv:builder
+      - image: helics/buildenv:ubuntu24.04-default-builder
     environment:
-      CMAKE_FLAGS: "-DHELICS_BUILD_CXX_SHARED_LIB=ON -DHELICS_BUILD_TESTS=ON -DHELICS_BUILD_BENCHMARKS=ON -DHELICS_INSTALL_PACKAGE_TESTS=ON"
+      CMAKE_FLAGS: "-DHELICS_BUILD_CXX_SHARED_LIB=ON -DHELICS_BUILD_TESTS=ON -DHELICS_BUILD_BENCHMARKS=ON -DHELICS_INSTALL_PACKAGE_TESTS=ON "
```

Comment:
Does the trailing space make a difference?

Reply:
probably not